ntbs-service/Models/ContactTracing.cs

```diff
@@ -43,5 +44,22 @@ public class ContactTracing : ModelBase
 
         [PositiveIntegerSmallerThanValue("ChildrenStartedTreatment", ErrorMessage = ValidationMessages.ContactTracingChildrenFinishedTreatment)]
         public int? ChildrenFinishedTreatment { get; set; }
+

```

Comment:
@jasiekmiko I remember talking about defining these on the individual models rather than Notification at some other point, but there was a reason not to do it... what has changed since?

Reply:
This were a few reasons
- It's nicer to understand which models owns this fields
- Total was not displayed when landing on Contact Tracing page. So, I just needed to display Total instead of 0 at the beginning.
- View is a bit cleaner (less chaining when accessing parameters)

Reply:
So I still think having them on `Notification` is nicer, as I think the consumer of these *doesn't* want to care about which sub-model they're coming from - they're all part of the notification record. I would argue having them on the `Notification` is how we get less chaining on the overview page (`Model.Total...` vs `Model.ContactTracing.Total...`).

But, that logic breaks apart once we start using these values on the individual pages, and it looks like we're using the `totals` this way here, as the last thing we'd want is to have them on both.

This leaves us in a bit of an unfortunate state, where we have most of them on Notification but some on the individual record, which could get confusing. Perhaps something to keep an eye on and see if we end up with more cases like this?